src/app/pages/Recipes/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import Footer from '../../components/Footer';
+import EditOutlinedIcon from '@mui/icons-material/EditOutlined';
+import DoneOutlinedIcon from '@mui/icons-material/DoneOutlined';
+import RecipesList from '../../components/RecipesList/RecipesList';
+import './index.css';
+
+export default (): JSX.Element => {
+  const [newRecipe, setNewRecipe] = useState<boolean>(false);
```

Comment:
newRecipe doesnt sound like a boolean so it is not so clear what this does.

Reply:
it opens up the form to add new recipe :D